yb-voyager/cmd/import.go

```diff
@@ -129,7 +129,7 @@ func registerFFDBAsTargetConnFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&tconf.Host, "ff-db-host", "127.0.0.1",
 		"host on which the Fall-forward DB server is running")
 
-	cmd.Flags().IntVar(&tconf.Port, "ff-db-port", -1,
+	cmd.Flags().IntVar(&tconf.Port, "ff-db-port", ORACLE_DEFAULT_PORT,
```

Comment:
Hmm, can we leave it as is maybe? because in future, we will have support for multiple FF DBs, right?

Reply:
Yes, exactly. And for now, only include oracle.